stan/math/prim/prob/ordered_logistic_rng.hpp

```diff
@@ -31,6 +31,28 @@ inline int ordered_logistic_rng(
   return categorical_rng(cut, rng);
 }
 
+template <
+    typename T_eta, typename T_c, class RNG,
+    require_t<disjunction<is_vector<T_eta>, is_std_vector<T_c>>>* = nullptr>
+inline std::vector<int> ordered_logistic_rng(const T_eta& eta, const T_c& c,
```

Comment:
Docs and also why is the vectorized implementation separate from the scalar one here? What was the problem?